src/rules/no-trailing-spaces.js

```diff
@@ -6,7 +6,7 @@ function noTrailingSpaces(parsedFile, fileName) {
   var lines = fs.readFileSync(fileName).toString().split('\n');
   var lineNo = 1;
   lines.forEach(function(line) {
-    if (line[line.length - 1] == ' ') {
+    if (/[\s]+$/.test(line)) {
```

Comment:
@guykisel  apparently there is a trimRight() but we can't straight up do:
`line.trimRight() === line` because `line` probably has a new line character at the end of it, which `trimRight()` gets rid off.

I realized that after this change we detect new lines as violators of the rule, so I am fixing that in: https://github.com/vsiakka/gherkin-lint/pull/49